schema/mysql/v57/temporal/schema.sql

```diff
@@ -245,19 +245,19 @@ CREATE TABLE cluster_metadata (
 
 CREATE TABLE cluster_membership
 (
+    membership_partition INT NOT NULL,
     host_id              BINARY(16) NOT NULL,
```

Comment:
can we just make this (host_id) a string? that seems like it would be easier to debug / look at the database? or is selecting a binary column via the mysql shell easy to read? Also gives more flexibility in the future so you aren't shoe-horned into a 16-byte identifier

Reply:
Context was 16-bytes is UUID size and this is almost more a session_id column as its provided on each restart. Id be a fan of having this change to session_id and have another field called host_id that could be user provided, but thatd be a more comprehensive change. Thoughts?